.github/workflows/cursor-pr-review-agent.yml

```
@@ -0,0 +1,277 @@
+          IS_BOT: ${{ steps.context.outputs.is_bot }}
+        run: |
+          SOURCE_REF='${{ github.event.pull_request.head.ref }}'
+          REVIEW_ID='${{ steps.context.outputs.review_id }}'
+          PR_NUMBER='${{ steps.context.outputs.pr_number }}'
+          echo "source_ref=$SOURCE_REF" >> $GITHUB_OUTPUT
+          echo "target_branch=$SOURCE_REF" >> $GITHUB_OUTPUT
+
+      - name: Build agent prompt
+        id: prompt
+        if: steps.comments.outputs.count != '0' && steps.context.outputs.is_bot != 'true'
+        run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2086:info:5:49: Double quote to prevent globbing and word splitting

(shellcheck)